tests/coverage/coverage.cpp

```diff
@@ -267,10 +267,67 @@ TEST(ImporterCoverage, importingComponentWithCnUnitsThatAreEmpty)
     model->addComponent(c);
 
     EXPECT_FALSE(model->hasUnresolvedImports());
+
+    validator->validateModel(model);
+    EXPECT_EQ(size_t(4), validator->errorCount());
```

Comment:
Would be nice to see what errors are expected here... and below at line 277.

Reply:
[d722583](https://github.com/cellml/libcellml/pull/1167/commits/d722583a6899b10ea5aa8f0bdcd5353f0177e056)